diff --git a/setup.py b/setup.py
index 2f99d3b..6d66bb4 100644
--- setup.py
+++ setup.py
@@ -580,7 +580,7 @@ class pil_build_ext(build_ext):
                 "PIL._webp", ["_webp.c"], libraries=libs, define_macros=defs))
 
         if feature.tcl and feature.tk:
-            if sys.platform == "darwin":
+            if sys.platform == "":
                 # locate Tcl/Tk frameworks
                 frameworks = []
                 framework_roots = [
